### PR TITLE
Moving MFA service into shared libs

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Moving MFA service into shared libs

#### Ticket Link
[MM-33007](https://mattermost.atlassian.net/browse/MM-33007)

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: utils/html.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func (t *HTMLTemplate) RenderToWriter(w io.Writer) error {
 	return nil
 }
 
-func TranslateAsHtml(t i18n.TranslateFunc, translationID string, args map[string]interface{}) template.HTML {
+func TranslateAsHTML(t i18n.TranslateFunc, translationID string, args map[string]interface{}) template.HTML {
```
Comment (agnivade) [inline utils/html.go:123]: Can we do this change in a separate PR? I'd like to keep this PR only for moving the service.

Comment (jespino) [reply utils/html.go:123]: Because the mattermost-utilities i18n extraction tool has changed to search for `TranslateAsHTML` this is needed to pass the CI's i18n-extract part.

Comment (jespino) [reply utils/html.go]: Master was already changing that, so... now is fixed anyway.

**Review by agnivade — APPROVED** [review]

**Review by mgdelacroix — APPROVED** [review]

**Review by lindy65 — APPROVED** [review]
Thanks @jespino - tested as per test steps on the Jira ticket and did not find any issues 👍
